src/components/WelcomeToSDC.tsx

```diff
@@ -20,6 +21,7 @@ type WelcomeToSDCProps = {
 
 export const WelcomeToSDC: React.FC<WelcomeToSDCProps> = ({ session }) => {
     const { t } = useTranslation("common");
+    const variant = useBreakpointValue({ base: "mobile", lg: "computer" });
```

Comment:
you got the idea here but can we maybe use a boolean, this seems difficult to keep track of
```suggestion
    const isMobileLayout = useBreakpointValue({ base: true, lg: false });
```